lib/semantics/resolve-names.cc

```diff
@@ -5118,7 +5118,7 @@ void ResolveNamesVisitor::CheckImports() {
     // C8102: all entities in host must not be hidden
     for (const auto &pair : scope.parent()) {
       auto &name{pair.first};
-      if (name != scope.name()) {
+      if (!scope.GetSymbol() || name != scope.name()) {
```

Comment:
This is a little too obscure.  It would be better to use the Symbol pointer returned from `GetSymbol()` and, when not null, test its name.  Or change `Scope::name()` to be a real accessor, actually -- it is named like one, and accessors should never crash; right now, it's dangerous.  Maybe it should be renamed and modified to return an optional result -- that seems best to me.